plugins/inputs/gtfs/README.md

```diff
@@ -0,0 +1,81 @@
+# GTFS
+
+This input plugin can be used to gather vehicle positions, trip updates and
+service alerts from a GTFS-realtime feed. [GTFS-realtime](https://developers.google.com/transit/gtfs-realtime) 
+is an extension of the [General Transit Feed Specification](https://developers.google.com/transit/gtfs).
+
+### Configuration
+
+This plugin can be configured to collect any combination of vehicle positions, trip updates
+and service alerts. Enable any one of these three input types by populating its URL in the
+plugin configuration. At least one of the three must be configured.
+
+```toml
+[[inputs.gtfs]]
+  ## API Key
+  # key = "${GTFS_API_KEY}"
```

Comment:
Does it mean "from environment"? If so, please add a comment above.

Reply:
It does, but that syntax shouldn't be in the example config. I will change it and comment accordingly.